cinema/views.py

```diff
@@ -118,7 +137,11 @@ class OrderPagination(PageNumberPagination):
     max_page_size = 100
 
 
-class OrderViewSet(viewsets.ModelViewSet):
+class OrderViewSet(
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin,
+    viewsets.GenericViewSet,
+):
     queryset = Order.objects.prefetch_related(
```

Comment:
Add permissions due to 
OrderViewSet - We should give the ability for authenticated users to create order